spec/integration/allow_inject_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+RSpec.describe "allow condition" do
+  describe "allow" do
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/gh/dry-rb/dry-auto_inject/pullRequest?prid=4763763)